### PR TITLE
Mobile CI Builds

### PR DESCRIPTION
**Pull request by thegoldenmule**
Automation for iOS and Android builds.

### CODE REVIEW

File: Assets/Editor/Build/UnityBuilderHooks.cs [hunk 1]
```diff
@@ -135,5 +160,23 @@ public static void RunTests()
                 target,
                 BuildOptions.None);
         }
+
+        [PostProcessBuildAttribute(1)]
```
Comment (gdamron) [inline Assets/Editor/Build/UnityBuilderHooks.cs]: comments!

File: build.ios.gradle [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+task runUnity(type:com.createar.plugins.unity.UnityMethodTask) {
+    method 'CreateAR.EnkluPlayer.Editor.UnityBuilderHooks.BuildIosPlayer'
+    target 'ios'
+    logPath 'ios.log'
+    unityBin UNITY_BIN
+    username UNITY_USERNAME
+    password UNITY_PASSWORD
+    serial UNITY_SERIAL
+}
+
```
Comment (gdamron) [inline build.ios.gradle]: a `runTests` task is conspicuously missing

File: Assets/Editor/Build/UnityBuilderHooks.cs [hunk 3]
```diff
@@ -85,13 +99,24 @@ public static void BuildWsaPlayer()
...
         {
             SwitchToIos();
 
             BuildPlayer(PATH_IOS, BuildTarget.iOS);
         }
 
+        /// <summary>
+        /// Builds app.
+        /// </summary>
+        [MenuItem("Tools/Build/Targets/Android")]
```
Comment (jsundra) [inline Assets/Editor/Build/UnityBuilderHooks.cs]: While in this file, maybe it'd be a good time to start moving some things to `Enklu/Tools/...` to be less ambiguous with Unity/other packages as we set our sights on a plugin?

File: build.ios.gradle (hunk 2 above)
Comment (thegoldenmule) [reply build.ios.gradle]: Yah... none of the `runTests` actually work.

Comment (gdamron) [reply build.ios.gradle]: ah, good to know

File: Assets/Editor/Build/UnityBuilderHooks.cs (hunk 3 above)
Comment (thegoldenmule) [reply Assets/Editor/Build/UnityBuilderHooks.cs]: There is an issue open for that, but I wasn't going to address it in this PR.

File: Assets/Editor/Build/UnityBuilderHooks.cs (hunk 1 above)
Comment (thegoldenmule) [reply Assets/Editor/Build/UnityBuilderHooks.cs]: :+1:

**Pull request merged**